content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -271,9 +271,9 @@ parameters:
   fsType: ext4
 ```
 
-* `type`: `io1`, `gp2`, `sc1`, `st1`. See
+* `type`: `io1`, `gp2`, `gp3`, `sc1`, `st1`. See
   [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html)
-  for details. Default: `gp2`.
+  for details. Default: `gp3`.
```

Comment:
How do we know this is the default _in the Kubernetes API_?

Reply:
Took the information from here => https://github.com/kubernetes-sigs/aws-ebs-csi-driver#createvolume-parameters

Reply:
I noticed that https://github.com/kubernetes-sigs/aws-ebs-csi-driver#createvolume-parameters is actually documenting a different (out of tree) storage driver.